plugins/talk-plugin-moderation-actions/client/constants.js

```diff
@@ -0,0 +1,4 @@
+const prefix = 'TALK_MODERATION_ACTIONS';
+
+export const OPEN_TOOLTIP = `${prefix}_OPEN_TOOLTIP`;
```

Comment:
According to wikipedia, a `tooltip `is sth that shows when hovering over it, so I don't think this is the correct term here. I'd just call it _menu_.